wordpress-mobile/WordPress-iOS
# [Site Name] Name screen - blank the search placeholder and focus search on load

Closes #18300

https://user-images.githubusercontent.com/2092798/162475546-1b601fcb-7235-46d9-8626-684cee987104.mp4

## To test:

Prerequisites: Access to the Site Intent and Site Name treatment groups. [See this PR for more information](https://github.com/wordpress-mobile/WordPress-iOS/pull/18298).

1. Start Site Creation flow
2. On the Site Intent view, tap a vertical or "Skip"
3. Expect the Site Name view to appear
4. Expect the input field to have focus
5. Expect there to be no placeholder text in the input field

## Regression Notes
1. Potential unintended areas of impact
- Site Name input field

6. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the above steps.

7. What automated tests I added (or what prevented me from doing so)
None.

PR submission checklist:
